packages/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -268,7 +269,8 @@ def change_internal_options(opt_path, pattern, value):
 
     with open(opt_path, "w") as sources:
         for line in lines:
-            sources.write(re.sub(f'{pattern}=[0-9]*', f'{pattern}={value}', line))
+            sources.write(
+                re.sub(f'{pattern}=[0-9]*', f'{pattern}={value}', line))
```

Comment:
I would not apply this change. If you want to improve readability create a variable above with the returned value of re.sub.

Reply:
Fixed. That line was modified by mistake. The newline there was not intended.